### PR TITLE
Greetings from alija

### PR DESCRIPTION
**Pull request by alijawosti**
hopefully fixing build failures

### CODE REVIEW

**Review by LinneaMay — COMMENTED** [review]
looks good!

**Review by LinneaMay — DISMISSED** [review]
looks good

File: src/main/java/hellos/Hellos.java
Comment (NicMcPhee) [inline src/main/java/hellos/Hellos.java]: This is very definitely a Bad Thing. You appear to have removed _many_ of the other people's greetings, probably while handling merge conflicts. You definitely removed all three of these, and it looks like you removed Ken, Pat, and Tristan below.

You have to fix that before you can merge this in, and no one should approve this PR without confirming that these have been restored.

**Review by NicMcPhee — CHANGES_REQUESTED** [review]
As mentionted above, you've removed (thrown away) a bunch of other people's greetings, which is totally not cool.

You have to fix that before you can merge this in, and no one should approve this PR without confirming that these have been restored.

**Pull request closed without merging**
